src/core_modules/capture-core/components/Pages/New/NewPage.component.js

```diff
@@ -84,10 +85,16 @@ const NewPagePlain = ({
             }
 
             {
-                newPageStatus === newPageStatuses.WITHOUT_PROGRAM_PARTNER_SELECTED &&
-                <InefficientSelectionsMessage
-                    message={i18n.t('Choose a category to start reporting')}
-                />
+                newPageStatus === newPageStatuses.WITHOUT_PROGRAM_CATEGORY_SELECTED &&
+                (() => {
+                    const { categoryCombination } = programCollection.get(currentScopeId) || {};
+                    const { name = 'program category' } = categoryCombination || {};
+                    return (
+                        <InefficientSelectionsMessage
+                            message={i18n.t('Choose a {{name}} to start reporting', { name })}
```

Comment:
A category combination can consist of mulitple categories. I think we should list the category names here instead of using the category combination name. We can discuss on slack or in our upcoming meeting.

Reply:
![image](https://user-images.githubusercontent.com/4181674/104012039-6c1e8c00-51af-11eb-8af7-4e654c03515c.png)
This looks like this but when for some reason `categoryCombination.name` is undefined then initial value of the name `program category`.